stubs/prettytable/prettytable/prettytable.pyi

```diff
@@ -141,6 +145,62 @@ class PrettyTable:
     def oldsortslice(self): ...
     @oldsortslice.setter
     def oldsortslice(self, val) -> None: ...
+    @property
+    def bottom_junction_char(self) -> Incomplete: ...
+    @bottom_junction_char.setter
+    def bottom_junction_char(self, val) -> None: ...
+    @property
+    def bottom_left_junction_char(self) -> Incomplete: ...
+    @bottom_left_junction_char.setter
+    def bottom_left_junction_char(self, val) -> None: ...
+    @property
+    def bottom_right_junction_char(self) -> Incomplete: ...
+    @bottom_right_junction_char.setter
+    def bottom_right_junction_char(self, val) -> None: ...
+    @property
+    def custom_format(self) -> Incomplete: ...
+    @custom_format.setter
+    def custom_format(self, val) -> None: ...
+    @property
+    def horizontal_align_char(self) -> Incomplete: ...
+    @horizontal_align_char.setter
+    def horizontal_align_char(self, val) -> None: ...
+    @property
+    def left_junction_char(self) -> Incomplete: ...
+    @left_junction_char.setter
+    def left_junction_char(self, val) -> None: ...
+    @property
+    def none_format(self) -> Incomplete: ...
+    @none_format.setter
+    def none_format(self, val) -> None: ...
+    @property
+    def preserve_internal_border(self) -> Incomplete: ...
+    @preserve_internal_border.setter
+    def preserve_internal_border(self, val) -> None: ...
+    @property
+    def right_junction_char(self) -> Incomplete: ...
+    @right_junction_char.setter
+    def right_junction_char(self, val) -> None: ...
+    @property
+    def top_junction_char(self) -> Incomplete: ...
+    @top_junction_char.setter
+    def top_junction_char(self, val) -> None: ...
+    @property
+    def top_left_junction_char(self) -> Incomplete: ...
+    @top_left_junction_char.setter
+    def top_left_junction_char(self, val) -> None: ...
+    @property
+    def top_right_junction_char(self) -> Incomplete: ...
+    @top_right_junction_char.setter
+    def top_right_junction_char(self, val) -> None: ...
+    @property
+    def xhtml(self) -> bool: ...
+    @xhtml.setter
+    def xhtml(self, val: bool) -> None: ...
+    @property
+    def rows(self) -> list[list[Any]]: ...
```

Comment:
Are you sure? It looks like a list of three-element tuples to me: https://github.com/jazzband/prettytable/blob/4345954111637455549bb7dab1e6f3714c32b0a6/src/prettytable/prettytable.py#L2489

Reply:
This is what I was looking at: https://github.com/jazzband/prettytable/blob/8ce5bd8956e12726f313db9644b03292400a49a7/src/prettytable/prettytable.py#L1414